src/pages/governance/_components/CircularCheckIcon.tsx

```diff
@@ -1,8 +1,16 @@
-export function CircularCheckIcon({ className }: { className: string }) {
+export function CircularCheckIcon({
+  className,
+  width,
```

Comment:
just fyi, you can also initialize at props level, feel free to resolve after you ack
```suggestion
  width = 40,
```